booster-android-gradle-v3_3/src/main/kotlin/com/didiglobal/booster/android/gradle/v3_3/V33.kt

```diff
@@ -159,6 +160,9 @@ internal object V33 : AGPInterface {
     override val BaseVariant.mergedRes: Collection<File>
         get() = getFinalArtifactFiles(InternalArtifactType.MERGED_RES)
 
+    override val BaseVariant.mergedNativeLibs: Collection<File>
+        get() = setOf((File("${project.buildDir.path}/${AndroidProject.FD_INTERMEDIATES}/transforms/mergeJniLibs/$name")))
```

Comment:
Please use `File.pathSeparator` instead of `\`

Reply:
use File.separatorChar to replace "\",please review again